test/database.test.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+
+var app = require('../server/index.js');
+var chai = require('chai');
+var request = require('supertest');
+var expect = chai.expect;
+
+describe('Content Service API Integration Tests', function() {
+    describe('#GET / games/search', function() {
+    it('should get all games', function(done) {
+      request(app) .get('/games/search')
+        .end(function(err, res) {
+          expect(res.statusCode).to.equal(200);
+          expect(res.body).to.be.an('array');
+          expect(res.body).to.be.empty;
+          done();
+        });
+        done();
+    });
+  });
+
+});
+
+// // Simple assertions
```

Comment:
Remove this if you don't plan on using this in the future.

Reply:
Done.